python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -1068,6 +1083,25 @@ def reset(self, errors_fatal=False):
                 record_local_head_state_if_needed(self.provider)
 
             self.available_node_types = self.config["available_node_types"]
+            if self.provider.dra_enabled():
+                if not self.dra_triggered_recently():
+                    self.dynamic_cpu = MIN_CPU_NUM
+                    self.dynamic_mem = MIN_MEM_SIZE
```

Comment:
Because we allow worker_conf["resources"]["CPU"] and worker_conf["resources"]["Memory"] to be None. Also,  because we have supported DRA, we should always make the idle Ray cluster to occupy minimal resource. The MIN_CPU_NUM and MIN_MEM_SIZE are representing the minimal resources (value less than that will cause a problem on vSphere).